cli/src/init.rs

```diff
@@ -0,0 +1,83 @@
+use crate::accounts::create_aze_game_account;
+use aze_lib::constants::{BUY_IN_AMOUNT, NO_OF_PLAYERS, SMALL_BLIND_AMOUNT};
+use aze_types::accounts::AccountCreationError;
+use clap::{Parser, ValueEnum};
+use figment::{
+    providers::{Format, Toml},
+    Figment,
+};
+use miden_objects::accounts::AccountId;
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(ValueEnum, Debug, Clone)]
+enum GameType {
+    Holdem,
+    Omaha,
+    Hilo,
+}
+
+#[derive(Debug, Clone, Parser)]
+pub struct InitCmd {
```

Comment:
Adding it here itself, We should also have a command to receive cards something like

`aze-cli peek-cards` Like the socket would log down when the game account has distributed all the cards.